canonical/ubuntu.com
# Fix links and fix menu padding

## Done

- Fixes links being the wrong colour on the nojs-nav and mobile menu
- Fixes the menu button padding being wrong when a secondary nav was present 

## QA

- Check that all links on the main dropdown, mobile navigation and nojs-navigation (/navigation) are #69c
- Check that when you are in a bubble ie. /desktop and on medium or small screens, the menu button has appropriate padding

## Issue / Card

Fixes 
https://warthogs.atlassian.net/browse/WD-7357
https://warthogs.atlassian.net/browse/WD-7358


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
